server_admin/topics/authentication/kerberos.adoc

```diff
@@ -1,239 +1,209 @@
+
 [[_kerberos]]
 
 === Kerberos
 
-{project_name} supports login with a Kerberos ticket through the SPNEGO protocol.
-SPNEGO (Simple and Protected GSSAPI Negotiation Mechanism) is used to authenticate transparently through the web browser after the user
-has been authenticated when logging-in his session.
-For non-web cases or when ticket is not available during login, {project_name} also supports login with Kerberos username/password.
+{project_name} supports login with a Kerberos ticket through the Simple and Protected GSSAPI Negotiation Mechanism (SPNEGO) protocol. SPNEGO authenticates transparently through the web browser after the user authenticates their session. For non-web cases, or when a ticket is not available during login, {project_name} supports login with Kerberos username and password.
 
 A typical use case for web authentication is the following:
 
-. User logs into his desktop (Such as a Windows machine in Active Directory domain or Linux machine with Kerberos integration enabled).
-. User then uses his browser (IE/Firefox/Chrome) to access a web application secured by {project_name}.
-. Application redirects to {project_name} login.
-. {project_name} renders HTML login screen together with status 401 and HTTP header `WWW-Authenticate: Negotiate`
-. In case that the browser has Kerberos ticket from desktop login, it transfers the desktop sign on information to the {project_name}
-  in header `Authorization: Negotiate 'spnego-token'` . Otherwise it just displays the login screen.
-. {project_name} validates token from the browser and authenticates the user.
-  It provisions user data from LDAP (in case of LDAPFederationProvider with Kerberos authentication support) or let user
-  to update his profile and prefill data (in case of KerberosFederationProvider).
-. {project_name} returns back to the application.
-  Communication between {project_name} and application happens through OpenID Connect or SAML messages.
-  The fact that {project_name} was authenticated through Kerberos is hidden from the application.
-  So {project_name} acts as broker to Kerberos/SPNEGO login.
-
-For setup there are 3 main parts:
-
-. Setup and configuration of Kerberos server (KDC)
-. Setup and configuration of {project_name} server
-. Setup and configuration of client machines
+. The user logs into the desktop.
+. The user accesses a web application secured by {project_name} using a browser.
+. The application redirects to {project_name} login.
+. {project_name} renders HTML login screen with status 401 and HTTP header `WWW-Authenticate: Negotiate`
+. If the browser has a Kerberos ticket from desktop login, the browser transfers the desktop sign-on information to the {project_name} in header `Authorization: Negotiate 'spnego-token'`. Otherwise, it displays the standard login screen, and the user enters their login credentials.
```

Comment:
remove the before {project_name}
To avoid plural "their" for a user and avoid "his or her"
Replace "screen, and the user enters their login credentials" with "screen that contains the username and password fields."